### PR TITLE
Update Spanish language file.

### PR DESCRIPTION
**Pull request by ricky-daniel13**
Added and translated missing strings.

### CODE REVIEW

File: HedgeModManager/Languages/es-MX.xaml [hunk 1]
```diff
@@ -25,7 +26,8 @@
     <system:String x:Key="MainUISave"          >Guardar</system:String>
     <system:String x:Key="MainUISaveAndPlay"   >Guardar y Jugar</system:String>
     <system:String x:Key="MainUIPlay"          >Jugar</system:String>
-
+    <system:String x:Key="MainUIMLDownloadFail">Descarga de Cargador de mods Fallida</system:String>
```
Comment (TProaa) [inline HedgeModManager/Languages/es-MX.xaml]:
```suggestion
    <system:String x:Key="MainUIMLDownloadFail">Descarga del cargador de mods fallida</system:String>
```

File: HedgeModManager/Languages/es-MX.xaml [hunk 2]
```diff
@@ -130,5 +134,10 @@
     <!-- Mod Config UI Strings -->
     <system:String x:Key="ModConfigUITitle">Configurar {0}</system:String>
 
+    <!-- Mod Downloader Strings -->
+    <system:String x:Key="ModDownloaderFailed">Descarga de Mod fallida</system:String>
```
Comment (TProaa) [inline HedgeModManager/Languages/es-MX.xaml]:
```suggestion
    <system:String x:Key="ModDownloaderFailed">Descarga del mod fallida</system:String>
```

**Review by TProaa — CHANGES_REQUESTED** [review]
I would suggest fixing the capitalization in 29 and in 138 so it doesn't look that off. And also changing the word "de" to "del" in both instances. Aside from that, solid translation.

**Comment by thesupersonic16** [comment]
LGTM

**Pull request merged**